app/src/components/inputs/inputOutside/inputOutside.scss

```diff
@@ -30,6 +30,7 @@
       background-repeat: no-repeat;
       background-size: 60%;
       background-position: center;
+      z-index: 2;
```

Comment:
Please follow our CSS [style guide](https://github.com/reportportal/service-ui/blob/v5/app/docs/8-CSS.md) for your style properties order